doxia-modules/doxia-module-markdown/src/test/java/org/apache/maven/doxia/module/markdown/MarkdownParserTest.java

```diff
@@ -340,6 +340,7 @@ public void testHtmlContent()
      * @param file the file to parse with {@link #parser}.
      * @return a sink to test parsing events.
      * @throws ParseException if the document parsing failed.
+     * @throws IOException if an I/O error occurs while closing test reader.
```

Comment:
no period at end of doc comment unless it's a complete sentence; per Oracle style guide

Reply:
I copied the existing precedent. Would you rather I change the other ones which were not affecting the build or make them inconsistent with one another?

Reply:
Up to you, but please don't introduce new violations

Reply:
Ok, will fix the others